packages/cli/src/number-parse.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Parses the input and returns it as a number if it's valid, otherwise throws error.
+ *
+ * @param {string} input
+ * @returns {number}
+ */
+export const parseNumber = (input = '') => {
+  const result = /[0-9]/.test(input) ? Number(input) : NaN;
```

Comment:
If we care about good error messages from `parseNumber(undefined)`:
```suggestion
export const parseNumber = input => {
  const result = /[0-9]/.test(input || '') ? Number(input) : NaN;
```